### DISCUSSION TITLE
Configurable system start for different areas of application.

### DISCUSSION THREAD

**Discussion by Banana71** [probonopd/MiniDexed#474] [Ideas]
I would find it fantastic to make the startup behavior configurable in the minidex.ini.

minidexed.ini
PowerOnMode: 0, default start to TG1 (as before)
PowerOnMode: 1, miniDexed start to Performance Load Menu
PowerOnMode: 2, E.Piano Mode, you can only choose between the Performances and do not make any changes.
PowerOnMode: 3, ?

In Mode 2, operating errors could be almost completely ruled out. (Children, Grandma, or live on Stage,....)

How do other synthesizers behave after a system start?

Yamaha SY99: returns exactly to the point at which the device was switched off.
Yamaha Motif ES: The behavior can be set in the Utility menu. (Performance, Voice, GM, last, Master)
Yamaha Montage, MODX: The behavior can be set in the Utility menu ("Live Set" or Performance)
The miniDexed is actually always in performance mode and starts with the TG1 menu. The only thing you can set so far is the default performance.

**Comment by Kirtai** [comment]
This might be better as an issue.
